### DISCUSSION TITLE
Не работает перенос файлов Wunjo AI

### DISCUSSION THREAD

**Discussion by Lukina-Tver** [wladradchenko/wunjo.wladradchenko.ru#24] [Q&A] [answered]
По ссылке https://github.com/wladradchenko/wunjo.wladradchenko.ru/wiki/%D0%9A%D0%B0%D0%BA-%D0%B8%D0%B7%D0%BC%D0%B5%D0%BD%D0%B8%D1%82%D1%8C-%D0%B4%D0%B8%D1%80%D0%B5%D0%BA%D1%82%D0%BE%D1%80%D0%B8%D1%8E-%D0%BF%D0%BE-%D1%83%D0%BC%D0%BE%D0%BB%D1%87%D0%B0%D0%BD%D0%B8%D1%8E-%D0%B4%D0%BB%D1%8F-%D0%BF%D0%B0%D0%BF%D0%BA%D0%B8-.wunjo описана опция, как в Windows перенести папку .wunjo в другое место.
Данная опция игнорируется. При исправлении файла C:\Program Files\Wladislav Radchenko\Wunjo Wladradchenko AI\app\backend\folders.py программа всё равно продолжает записывать файлы в папку c:\Users\User\.wunjo\
Ниже приведено содержимое нашего файла folders.py
----------------
import os

os.environ["HOME"] = "N:\Wunjo"
HOME_FOLDER = os.path.expanduser('~')

MEDIA_FOLDER = os.path.join(HOME_FOLDER, '.wunjo')
if not os.path.exists(MEDIA_FOLDER):
    os.makedirs(MEDIA_FOLDER)

SETTING_FOLDER = os.path.join(MEDIA_FOLDER, 'setting')
if not os.path.exists(SETTING_FOLDER):
    os.makedirs(SETTING_FOLDER)

WAVES_FOLDER = os.path.join(MEDIA_FOLDER, 'waves')
if not os.path.exists(WAVES_FOLDER):
    os.makedirs(WAVES_FOLDER)

AVATAR_FOLDER = os.path.join(MEDIA_FOLDER, "avatar")
if not os.path.exists(AVATAR_FOLDER):
    os.makedirs(AVATAR_FOLDER)

DEEPFAKE_FOLDER = os.path.join(MEDIA_FOLDER, 'video')
if not os.path.exists(DEEPFAKE_FOLDER):
    os.makedirs(DEEPFAKE_FOLDER)

DEEPFAKE_MODEL_FOLDER = os.path.join(MEDIA_FOLDER, 'deepfake')
if not os.path.exists(DEEPFAKE_MODEL_FOLDER):
    os.makedirs(DEEPFAKE_MODEL_FOLDER)

VOICE_FOLDER = os.path.join(MEDIA_FOLDER, 'voice')
if not os.path.exists(VOICE_FOLDER):
    os.makedirs(VOICE_FOLDER)

RTVC_VOICE_FOLDER = os.path.join(MEDIA_FOLDER, 'rtvc')
if not os.path.exists(RTVC_VOICE_FOLDER):
    os.makedirs(RTVC_VOICE_FOLDER)

CUSTOM_VOICE_FOLDER = os.path.join(MEDIA_FOLDER, 'user_trained_voice')
if not os.path.exists(CUSTOM_VOICE_FOLDER):
    os.makedirs(CUSTOM_VOICE_FOLDER)

TMP_FOLDER = os.path.join(MEDIA_FOLDER, 'tmp')
if not os.path.exists(TMP_FOLDER):
    os.makedirs(TMP_FOLDER)

**Comment by wladradchenko** [comment, marked as answer]
Привет.

Причина по которой не работает, потому что пишется: `N:\\Wunjo`. Если все равно не заработает, попробуйте `os.environ["USERPROFILE"]="N:\\Wunjo"`. Если вариант с HOME не работает, но работает только USERPROFILE, напишите мне, чтобы я добавил соответствующие изменения в документацию для пользователей Windows

**Reply by Lukina-Tver** [reply]
Получилось только с os.environ["USERPROFILE"]="N:\\Wunjo". Можете исправлять документацию, только не забудьте, пожалуйста, указать, что в пути к папке должны быть 2 косые черты, а не одна.
